code/client/src/core/ui/world_debug.cpp

```diff
@@ -244,6 +248,68 @@ namespace MafiaMP::Core::UI {
             }
         }
 
+        if (ImGui::CollapsingHeader("Racing")) {
+            using namespace SDK::mafia::ui;
+            using namespace SDK::mafia::database;
```

Comment:
We should avoid this, it's always better to use the fully-fledged SDK namespace path on mod side